src/badguy/ghoul.hpp

```diff
@@ -0,0 +1,65 @@
+//  SuperTux
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef HEADER_SUPERTUX_BADGUY_GHOUL_HPP
+#define HEADER_SUPERTUX_BADGUY_GHOUL_HPP
+
+#include "badguy/badguy.hpp"
+#include "object/path_object.hpp"
+
+class Ghoul final : public BadGuy,
+                    public PathObject
+{
+public:
+  Ghoul(const ReaderMapping& reader);
+  virtual std::string get_class() const override { return "ghoul"; }

```

Comment:
I do not think we want to subclass Ghoul at any point. Please remove the "virtual" specifier from all the methods.

Reply:
I think you need to keep the `override`s
https://travis-ci.org/SuperTux/supertux/jobs/590824166#L3969